### PR TITLE
Link to `Client/Release stream/Lazer` from many different articles

### PR DESCRIPTION
**Pull request by TicClick**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- `SKIP_WIKILINK_CHECK` (not relevant to my changes)

### CODE REVIEW

File: wiki/Community/osu!_community_meetings/zh-tw.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ outdated_since: 4a75f667505af5dc619a81b7326eac73d6483a3e
 
 | 會議編號 | 連結 | 摘要 | 議程 |
 | :-: | :-- | :-- | :-- |
-| 0 | [2021 年 9 月 19 號](https://youtu.be/-LjNll8VYTM) | [筆記](https://gist.github.com/peppy/08b38cbac56a8b1e88133853a6a8d2fb) | 大致介紹此會議、審核標準、難度及 pp、osu!(lazer) |
+| 0 | [2021 年 9 月 19 號](https://youtu.be/-LjNll8VYTM) | [筆記](https://gist.github.com/peppy/08b38cbac56a8b1e88133853a6a8d2fb) | 大致介紹此會議、審核標準、難度及 [osu!(lazer)](/wiki/Client/Release_stream/Lazer) |
```
Comment (Walavouchey) [inline wiki/Community/osu!_community_meetings/zh-tw.md:37]: pp removed?

File: wiki/Contests/triangles_Beatmapping_Contest/zh.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ no_native_review: true
 
 ![](https://assets.ppy.sh/contests/147/header.jpg)
 
-**三角作图比赛**是以激光版主题曲 *[cYsmix](https://osu.ppy.sh/beatmaps/artists/2) - triangles* 为焦点的作图和故事板设计比赛。
+**三角作图比赛**[是以激光版](/wiki/Client/Release_stream/Lazer)主题曲 *[cYsmix](https://osu.ppy.sh/beatmaps/artists/2) - triangles* 为焦点的作图和故事板设计比赛。
```
Comment (Walavouchey) [inline wiki/Contests/triangles_Beatmapping_Contest/zh.md:9]: should be untranslated like everywhere else

**Review by Walavouchey — COMMENTED** [review]
good stuff

**Review by peppy — APPROVED** [review]
Looks solid

**Pull request merged**